
<html class="no-js" lang="">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="x-ua-compatible" content="ie=edge">
        <title>Biosystems Research UCSF</title>
        <link rel="icon" type="images/png" href="img/favicon.ico">
        <meta name="description" content="">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <link rel="apple-touch-icon" href="apple-touch-icon.png">
        <!-- Place favicon.ico in the root directory -->

        <link rel="stylesheet" href="css/normalize.css">
        <link rel="stylesheet" href="css/main.css">
        <!-- Latest compiled and minified CSS -->
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">

        <!-- Optional theme -->
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css" integrity="sha384-fLW2N01lMqjakBkx3l/M9EahuwpSfeNvV63J5ezn3uZzapT0u7EYsXMjQV+0En5r" crossorigin="anonymous">
        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
        <link rel="stylesheet" href="css/research.css">
      </head>
      <body>
        <!--[if lt IE 8]>
                <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
            <![endif]-->

        <!-- Add your site or application content here -->
        <nav class="navbar navbar-inverse">
          <div class="container-fluid">
      <!-- Brand and toggle get grouped for better mobile display -->
      <div class="navbar-header">
        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
          <span class="sr-only">Toggle navigation</span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>
        <a class="navbar-brand" href="index.html">Biosystems Group at UCSF</a>
      </div>

      <!-- Collect the nav links, forms, and other content for toggling -->
      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
        <ul class="nav navbar-nav">
          <li><a href="index.html">Home</a></li>
          <li class="active"><a href="research.html">Research<span class="sr-only">(current)</span></a></li>
          <li><a href="people.html">People</a></li>
          <li><a href="#">Collaboration</a></li>
          <li><a href="#">Links</a></li>
          <li><a href="#">Observations</a></li>
          <li><a href="#">Publications</a></li>
          </li>
        </ul>
      </div><!-- /.navbar-collapse -->
    </div><!-- /.container-fluid -->
  </nav>
          <div class="container-fluid">
            <div class="row blackHeader">
              <div class="col-xs-12">
                <img src="img/unpolarizedvspolarized.png" alt ="Cell diagram of polarized and unpolarized" class="responsive">
              </div>
            </div>
            <div class="row">
              <div class="col-xs-12">
                <h3>Current Interdisciplinary Research</h3>
              </div>
            </div>
            <div class="row researchArticles">
              <div class="col-xs-12">
                <div class="row articleBlock">
                  <div class="col-xs-12 specialSection3">
                    <h4 class="specialHeader3">Dictionary for Modeling and Simulation in Biological Systems</h4>
                    <div class="col-xs-12 col-sm-4 specialText3">
                    <!---Use the search html thing Ifound-->
                    <div class="input-group">
                      <span class="input-group-btn">
                        <button class="btn btn-default" id="dictionarySearch" type="button">Go!</button>
                      </span>
                      <input type="text" class="form-control" id="userInput" method="" placeholder="Search Dictionary...">
                    </div>
                    <!-- <div class="searchResultDiv"></div> -->
                  </div>
                  <div class="col-xs-12 col-sm-8 specialText3">
                    <!--Create with javascript event listener-->
                    <!--I want links on hover to get bigger-->
                    <ul class="alphabetLinks">
                      <li class="a"><a href="#">A</a></li>
                      <li class="b"><a href="#">B</a></li>
                      <li class="c"><a href="#">C</a></li>
                      <li class="d"><a href="#">D</a></li>
                      <li class="e"><a href="#">E</a></li>
                      <li class="f"><a href="#">F</a></li>
                      <li class="g"><a href="#">G</a></li>
                      <li class="h"><a href="#">H</a></li>
                      <li class="i"><a href="#">I</a></li>
                      <li class="l"><a href="#">L</a></li>
                      <li class="m"><a href="#">M</a></li>
                      <li class="o"><a href="#">O</a></li>
                      <li class="p"><a href="#">P</a></li>
                      <li class="r"><a href="#">R</a></li>
                      <li class="s"><a href="#">S</a></li>
                      <li class="t"><a href="#">T</a></li>
                      <li class="u"><a href="#">U</a></li>
                      <li class="v"><a href="#">V</a></li>
                      <li class="w"><a href="#">W</a></li>
                    </ul>
                  </div>

                <div class="col-xs-12 openDictionary specialText3">
                </div>
                </div>
              </div>

                <div class="row articleBlock">
                  <div class="col-xs-12 list-group">
                    <h4 class="list-group-item-heading">Sinusoidal Segment Interactive 3D Visualization</h4>
                    <div class="col-xs-12 list-group-item-text">
                      <img class="smallMedia" src="img/sinusoidalSegment.gif" alt="Sinusoidal Segment Still">
                      <h5>Description</h5>
                      <p>Executable file is available upon request.</p>
                      <p>This movie is an interactive 3D visualization of antipyrine & sucrose compounds percolating through one &#40;&#35;7&#41; out of about 60&#41; Sinusoidal Segment in the In Silico Liver &#40;over 400 time steps&#41;. The three grid spaces &#40;pink: sinusoidal, yellow: endothelial, and blue: space of the space of Disse plus hepatocytes&#41; are rendered as concentric cylinders. Two views are provided of these cylinders. One is a side view &#40;upper right&#41; and the other is a head-on view &#40;lower left&#41;. At the lower right the raw grid data is directly represented.<br>For each grid space point there are several options&#58;</p>
                      <ul class="sinusoidalList">
                        <li>It can be empty, that is, no cell there &#40;rendered as fully transparent&#41;</li>
                        <li>It can contain an empty cell &#40;rendered as mostly transparent white&#41;</li>
                        <li>It can contain antipyrine or sucrose particles outside of a cell &#40;red or green&#41;</li>
                        <li>It can contain a cell with one or more free antipyrine particles within it.</li>
                        <li>It can contain a cell with one or more bound antipyrine particles within it. For example: &#34;3 bound&#34; means 3 bound antipyrine particles in a cell.</li>
                      </ul>
                      <p>Each &#34;smooth&#34; control on the upper left is used to distinguish the spaces; it causes the grid data to be blurred or not. The blurring is being applied as a texture to a cylinder and could be viewed as representing the likelihood that a compound of that type is actually in that location.<br><br>
                      The &#34;Show&#34; control selects the space&#45;sinusoidal, Endothelial and Space of Disse &#38; Hepatocytes &#40;SoD&#41; to be observed &#40;or not&#41;.<br><br>
                      Selecting &#34;Rotate&#34; causes the cylinder to rotate.<br><br>
                      The &#34;&#62;||&#34; &#34;||&#60;&#34; controls causes in silico time to move one step forward or backward.<br>Notes&#58;</p>
                      <ul class="sinusoidalList">
                        <li>Compounds are injected into the In Silico Liver using a tight distribution function&#59;</li>
                        <li>Antipyrine, but not sucrose, can enter cells&#59;</li>
                        <li>There are objects within &#34;cells&#34; that can bind antipyrine&#59;</li>
                        <li>Antipyrine bound within &#34;hepatocytes&#34; in the Disse layer can be metabolized.</li>
                        <li>Metabolites of antipyrine are not visualized&#59;</li>
                        <li>In this movie there is no more than one free solute per cell.</li>
                      </ul>
                      <p>To view the movie you need <a href="https://get.adobe.com/shockwave/" target="_blank"> Macromedia Shockwave Player 10</a> running on a late-model computer &#40;Mac or Win&#41; with openGL and hardware 3D acceleration.<br><br>For additional biological and pharmacokinetic detail see&#58;<br><br>Hung DY, Chang P, Weiss M, Roberst MS &#40;2001&#41;  <a href="http://jpet.aspetjournals.org/content/297/2/780.full.pdf+html" target="_blank">Structure&#45;hepatic disposition relationships for cationic drugs in isolated perfused rat livers&#58; transmembrane exchange and cytoplasmic binding process.</a> J Pharmacol Exp Ther. 297&#58;780&#45;9.</p>

                    </div>
                </div>
              </div>
              <div class="row articleBlock">
                  <div class="col-xs-12 list-group">
                    <h4 class="list-group-item-heading">Functional Unit Representation Model Project</h4>
                    <div class="col-xs-12 list-group-item-text">
                      <p>
                        FURM is a modeling method for building a multitier, in silico apparatus to support iterative experimentation on multiple models.
                      <br>The development of any given biological model is driven by the experimental context in which it will be used. Hence, computer models are often over-fitted to a single, unique, experimental context and fail to be useful in other situations. To solve this problem, multiple, separate models of a biological system are required to adequately represent that system's behaviors. This is in stark contrast to the biological components to which these models refer, which can be placed in almost any experimental context relevant to their various behaviors. The problem presents a fundamental breakdown in the extent to which any computer model represents its referent. We present the basics of a modeling method, FURM &#40;Functional Unit Representation Method&#41; that attempts to address this breakdown by enforcing and encouraging the use of some basic methodological principles in the development of any functional unit model.</p>
                      <h5><a href="http://furm.org" target="_blank">Learn More</a></h5>
                    </div>
                  </div>
                </div>

                <div class="row articleBlock">
                  <div class="col-xs-12 list-group">
                    <h4 class="list-group-item-heading">Relating GeneChip® Data To Measures of Pharmacological and Toxicological Phenotypes</h4>
                    <div class="list-group-item-text">
                      <p>
                      Can one use microarray measured gene expression &#40;MMGE&#41; data to predict pharmacological and toxicological phenotypes in animals and man? Can similar data be used to account for a significant portion of the individualized variability in patient drug response? In collaboration with the Computational Biology Group at <a href="#" target="_blank">Roche Palo Alto</a>, we are evaluating and contrasting the computational approaches that have been proposed to address these questions. Two large sets of MMGE data are being used to compare and contrast the information that results from use of the various available statistical pattern extraction, clustering and evaluation tools and techniques. Our current focus is on a variable selection and testing strategy that is particularly driven by directly relevant archival phenotypic data, information and knowledge.<br><br>Key personnel&#58;  <a href="#">Yuanyuan Xiao</a> and <a href="#">Donglei Hu.</a><br><br>Collaborators&#58; Mark Segal, Steve Shiboski, Michael N. Liebman, and Roche Palo Alto.</p>
                  </div>
                </div>
              </div>
                <div class="row articleBlock">
                  <div class="col-xs-12 list-group">
                    <h4 class="list-group-item-heading">Toward Genotype-Phenotype Linkage to Optimize Cancer Treatments</h4>
                    <div class="list-group-item-text">
                      <p>
                      This project is part of a larger undertaking that will assemble, test and validate a prototype decision support framework &#40;DSF&#41;. We are using both informatics and simulation strategies to create a &#34;mirror&#34; population of hypothetical patients that enables linkage of the information from highly multiplexed molecular analyses through available systems knowledge to patient clinical data so that expected changes in treatment outcomes for new patients can be visualized &#40;on a computer&#41; interactively as various options and/or new data are considered. The DSF is intended to function as part of an institution-wide, patient centered system. With such a system, the patient, the physician, and researchers will be able to detect important, decision impacting relationships contained within the cancer research and clinical trial databases relationships that otherwise might remain undetected. The DSF will also allow the decision makers, the patient and physician, to ask a variety of probing &#34;what if&#34;? questions, and to visually compare and contrast the answers graphically represented, with expected treatment results in real time. Having such a capability will allow customized &#40;and, ideally, optimized&#41; individual treatments, thereby lowering costs while improving both outcomes and the patient's quality of life.<br><br>
                      Project contact&#58; C. Anthony Hunt<br><br>
                      Collaborators&#58; <a href="http://www.oztechsys.com" target="_blank">Oztech Systems</a>, <a href="http://www.msa.com" target="_blank">Management Sciences Associates</a>, Dan H. Moore, Joe W. Gray
                    </p>
                  </div>
                  </div>
                </div>
                <div class="row articleBlock">
                  <div class="col-xs-12 list-group">
                    <h4 class="list-group-item-heading">A Patient-Centered Decision Support Framework for Breast Cancer</h4>
                    <div class="list-group-item-text">
                      <p>
                        This project is a breast cancer specific realization of the project &#34;Toward Genotype Phenotype Linkage to Optimize Cancer Treatments.&#34; How can emerging genetic and specific molecular information on breast cancer progression be linked to outcomes associated with different treatments, so that the individual needs of the patient physician interaction can be optimally coordinated? There are critical gaps between the collection of biomedical information &#40;biological and medical informatics&#41; and its most useful application for impacting selection of treatment options. Our goal is to provide an integrated informatics system focused on the key issues in breast cancer treatment that places decision making power in the hands of the clinician, while providing outcome probabilities for the options available to the patient. The DSF &#40;decision support framework&#41; design anticipates making optimum use of such emerging technologies as microarray measures of gene expression. The immediate goal is to demonstrate the feasibility of a DSF that will enable systematic individualization of breast tumor treatments. A computational database is being developed that represents a &#34;mirror&#34; population of breast cancer patients. The core experimental database includes over two dozen prognostic &#40;i.e., breast cancer markers, such as ER status, Her2, etc.&#41;, risk, medical, and key patient factors for several hundred patients, plus results from recent pharmacogenetic clinical trials. Each new patient can be matched with a small set &#34;surrogates&#34; within the DSF database, which allows one to ask many &#34;what if&#634;&#34; questions about treatment options and possible outcomes. The next step is to apply and develop algorithms to predict uncertainty throughout the decision making process. In part, our effort is similar to the application of Bayesian Networks to breast cancer. The project anticipates a future clinical setting where breast tumor treatments can be individually optimized, errors will diminish, outcomes will improve, and there will be considerable cost savings for the individual and for society.<br><br>
                        Key personnel&#58;  Christine Case<br><br>
                        Collaborators&#58; John W. Park, Laura J. Esserman, Laurence H. Baker, and Dan Moore
                    </p>
                  </div>
                  </div>
                </div>
                <div class="row articleBlock">
                  <div class="col-xs-12 list-group">
                    <h4 class="list-group-item-heading">Using Stochastic Activity Network Simulations to Better Understand Disorders of Hemostasis</h4>
                    <div class="list-group-item-text">
                      <p>
                        Stochastic Activity Networks &#40;SAN&#41; may have advantages over traditional modeling approaches for understanding the complex processes. We are using this tool to better understand the interacting biochemical events involved in the coagulation and fibrinolysis processes that cause Disseminated Intravascular Coagulation! &#40;DIC&#41;. We use the UltraSAN2 software environment to simulate these processes. Within the SAN we can inactivate or alter certain proteins to mimic genetic defects and disease, and then compare the results to clinical observations for validation. We are developing this simulation project through 7 steps: 1&#41; trypsinogen activation and trypsin inactivation, 2&#41; normal blood coagulation, 3&#41; disorders of blood coagulation, 4&#41; normal fibrinolysis, 5&#41; disorders of fibrinolysis, 6&#41; normal hemostasis, and 7&#41; DIC. Results of the successful simulations of steps 1&#63;3 by Mounts and Liebman &#91;1&#93;3 provides a promising foundation. A combination of the SAN that separately represents of steps 2 and 4 is expected to adequately represent normal hemostasis. We will systematically inactivate sets of proteins in the final, linked SAN to generate results that are consistent with the clinical observations of DIC. The final, linked SAN will be used to assist in the identification of new therapeutic targets, suggest strategies for development of new diagnostics, and provide a network hypothesis for connecting to emerging genetic information.<br><br>
                        Key personnel&#58;  Keith Erickson<br><br>
                        Collaborators&#58; Michael Liebman and Adam P. Arkin<br><br>
                        Mounts, W.M. and Liebman, M.N. Qualitative Modeling of Normal Blood Coagulation and its Pathological States Using Stochastic Activity Networks. Intl J of Biological Macromolecules 20:265-281 &#40;1997&#41;.
                    </p>
                  </div>
                  </div>
                </div>
                <div class="row articleBlock">
                  <div class="col-xs-12 list-group">
                    <h4 class="list-group-item-heading">Elucidating Critical Elements Within Pharmacogenetic and Biomolecular Pathways</h4>
                    <div class="list-group-item-text">
                      <p>
                    We are systematically evaluating the applicability of graph theoretic techniques to facilitate utilization of large pharmacological, genetic and physiological system models. The conceptual or computational model &#40;such the above Stochastic Activity Networks&#41; in is first represented as a network graph having nodes and edges with respective weights. Computational graph analysis is then used to bring critical elements into focus by grouping closely related system elements prior to finding and ranking elements or paths between elements that have the greatest influence on the system. When applied to pharmacological and physiological system models, such "critical elements" represent potential drug targets. By modulating the function of these elements or specific interactions between elements, we expect to be able to identify the process or event where intervention is most likely to have the greatest effect on potential outcome functions, such as drug or treatment effectiveness or undesired side effects.<br><br>
                    Key personnel&#58; <a href="#">Abraham Anderson</a><br><br>
                    Collaborators&#58; Richard Ho, Director of Medical Informatics, <a href="#" target="_blank">R.W. Johnson Pharmaceutical Research Institute</a>
                    </p>
                  </div>
                  </div>
                </div>
                <div class="row articleBlock">
                  <div class="col-xs-12 list-group">
                    <h4 class="list-group-item-heading">Pharmacogenetics And Individual Variability: Simulation Using Agent-Based Programming.</h4>
                    <div class="list-group-item-text">
                    <p>  There is a critical need for new innovative approaches to understanding and modeling interindividual variability and how it impacts development of new therapeutic strategies, e.g., new drugs, as well as the utilization of existing therapeutic strategies. Individual patient variability, such as cardiovascular status, pharmacogenetics, etc. are known to contribute to pharmacodynamic variability. Such variability is difficult to capture using traditional modeling strategies. We are successfully using a novel approach that uses an agent-based programming platform &#40;e.g., <a href="http://www.swarm.org/wiki/Main_Page" target="_blank">SWARM.</a>&#41; The approach can provide an in silico patient population that responds appropriately to drugs based on individual patient attributes. While current approaches in the field rely heavily on complex, often stiff statistical models that are drug specific, our in silico approach offers the ability to model random biological fluctuations inherent in human populations along with a program which may be easily modified to test an array of therapeutic products. The long-term and potential groundbreaking benefits of such an undertaking include targeting appropriate populations for drug safety and efficacy trials, and having a tool that can be used to optimize drug regimens for specific patients.<br><br>
                      Key personnel&#58; Amina A. Qutub<br><br>Collaborators&#58; Alex Lancaster
                    </p>
                  </div>
                </div>
                </div>
                <div class="row articleBlock">
                  <div class="col-xs-12 list-group">
                    <h4 class="list-group-item-heading">A Method for Visualizing Similarity in Gene Expression Datasets</h4>
                    <div class="list-group-item-text">
                      <p>We made significant progress in developing a method of mapping large datasets onto a two dimensional surface where every point corresponds to an Iterated Function Systems &#40;IFS&#41;. These IFS can be used to generate an long sequence of scalable arbitrary numbers. There is a correspondence between the similarity of these sequences and their location on the surface of the map. The values contained in these generated sequences is compared against sequential values of target datasets. A point that corresponds to the "best fit" to each target dataset is then located. One can quantify the relationship between similarities of target datasets and the location and geometry of their corresponding points on such a map with a view to optimizing the technique for visualizing relative similarity &#40;or difference&#41; between large sets of gene expression data. Sandy Shaw leveraged the progress made into a new company, Fractal Genomics.<br><br>
                        Key personnel&#58; Sandy Shaw<br><br>Collaborators&#58; <a herf="https://math.berkeley.edu/~harrison/Home.html" target="_blank">Jenny Harrison</a> &#40;Math UCB&#41;.
                    </p>
                  </div>
                  </div>
                </div>
                <div class="row articleBlock">
                  <div class="col-xs-12 list-group">
                    <h4 class="list-group-item-heading">Microarray Gene Expression In Support of Immunomodulator Discovery</h4>
                    <div class="list-group-item-text">
                      <p>Can one prolong graft survival in transplant patients by using novel therapeutics oligonucleotides to decrease the ability of the graft to trigger the host immune system. Such a strategy is in contrast to the more common approach involving host immunosuppression. We are investigating a series of oligonucleotides that were designed to downregulate the expression of MHC class I and class II genes. In particular, we are interested in the extent to which constitutive and induced expression of MHC genes can be reduced by therapeutic oligonucleotides, and the resultant effect on the host immune response. In Project One, the Hunt Group is implementing strategies to acquire, evaluate, and use microarray measured gene expression to facilitate the above approach.<br><br>
                        Collaborators&#58;Richard Shafer
                    </p>
                  </div>
                  </div>
                </div>
                <div class="row articleBlock">
                  <div class="col-xs-12 specialSection">
                    <h4 class="specialHeader">Swarm Modeling and Simulation</h4>
                    <div class="specialText">
                      <h5>Phase 1</h5>
                      <h5>Simple Swarm Simulation for Cancer Spheroid</h5>
                      <ul  class="nav nav-tabs" roll="tablist">
                        <li role="presentation" class="active"><a href="#mayMinutes" aria-controls="mayMinutes" role="tab" data-toggle="tab">5-30-02 Meeting Minutes</a></li>
                        <li id="june" role="presentation"><a href="#juneMinutes" aria-controls="juneMinutes" role="tab" data-toggle="tab">6-3-02 Meeting Minutes</a></li>
                      </ul>
                      <div class="tab-content">
                        <div role="tabpanel" class="tab-pane active" id="mayMinutes">
                          <h5>Meeting Minutes</h5>
                          <p>31 May 2002<br>
                            11&#58;00 a.m. &#45; 1&#58; p.m. HSE&#45;160<br>
                            Cindy Li and Amy H attending<br><br>
                            We began the meeting by discussing a project web page. Cindy agreed to take care of the web page, and we briefly discussed password protecting its contents before deciding against doing so.<br><br>
                            We turned our attention to what properties the cells of the tumor should have. Li suggested that we consider oxygen pressure and the area and density of the tumor and create a schedule of objects. Cindy then expressed a desire for a more "top-down" approach, and so we talked about what we wanted the system to do. We agreed that we wanted to simulate growth using a constant growth rate. For the present, we will ignore quiescent and necrotic populations and cell death. I agree it is a good idea to get a simple working program first, even if you know that lots of changes will need to be made. Later, you’ll need to define different cell states, and rules for how they influence each other’s behavior. We also decided that there should be a certain size or number of cells comprising a tumor at which growth ceases, Later you will need to remove this condition. There is no evidence that any one cell is aware of the size &#40;&#35;&#41; of the cell mass that contains it. but we could not reach a consensus on whether size or number of cells should be used. Cells should divide when there is enough nutrient available and at either a certain time or a certain size &#40;we could not reach a consensus on this topic, and it is one that we debated for quite some time&#41;. Amy H. also introduced the idea of random cell sizes or random dividing times &#40;e.g., if these conditions are met, then Prob. &#61; p that division will initiate at the start of the step&#41; as having all the cells in the tumor divide at the same time seemed artificial.<br><br>
                            At this point we believed that having two classes, a class for cells and a class for the environment, would be the best option, and we attempted to determine rules and properties for the cells and the environment. However, we disagreed about if there should be “levels” for the cells. Cindy thought that the cells and the environment should be on the same level and that their interactions would be mediated by something on a higher level &#40;Is there evidence to support this position?&#41;. Amy H. and Li expected that the cells should make “decisions” on their own without such an upper level. Each of us then drew out her own idea of the structure of the program. The concept behind Li’s program was that if there was sufficient amounts of nutrient &#40;N>Nd&#41; and sufficient cell size &#40;S=2S0&#41; but the tumor itself was small enough &#40;it hadn’t reached its limiting size&#41; that a cell would consume some nutrient &#40;Nd&#41; and split into two cells of size S0. Microscopy on spheroids tends to show that daughter cells tend to be smaller after division. They then increase to adult size. Once a daughter cell reaches adult size, there is only a small change in cell size as it then moves through cell the cell cycle. Amy H. questioned the availability of nutrient to a cell at a particular location, and the group agreed to assume that nutrient was freely diffusing through the environment rather than being concentrated at some point.<br><br>
                            Cindy still had some concerns about the plan of attack, so we tried to come up with properties and rules that the cells and the environment would have or follow. It became clear that it would be challenging to separate the rules for the environment and the cells. For example when a cell consumes some nutrient, it affects both the cell’s size &#40;it grows&#41; and the amount of nutrient left in the environment. How could we account for both of these consequences? We experimented with the following: if N ≥ Nd, then S&#61;2S0 &#40;the cell would immediately grow to division size&#41; and N&#61;N-Nd. This plan only allows for the splitting of one cell though. Li then suggested that we eliminate the environment class and instead have a fixed amount of nutrient present only as a parameter. However, this did not avoid the problem of how much nutrient is consumed at a particular time step. Important point We realized that we needed to have a count of the cells to determine how much total nutrient is consumed. Our next iteration: at t&#61;0, there are C0 cells and N nutrients. If N ≥ C0Nd, then C0&#61;2C0 &#40;cells split&#41; and N&#61;N-C0Nd &#40;each cell consumes Nd nutrient&#41;. If N < Nd, stop. Amy H. pointed out that this strategy left nutrients unused, an unlikely scenario, and Li modified by adding another loop: if Nd ≤ N < C0Nd, C1 &#61; int&#40;N/ Nd &#41; for some integer C1, so C0&#61; C0+C1 &#40;the C1 cells split&#41;. Cindy proposed a different approach where each cell checks the availability of resources &#40;nutrients&#41; for itself only. In other words, if N≥Nd, cell 1 splits. Then if N≥Nd, cell 2 splits, etc. Li thought this approach impractical as cells can split at the same time, but Cindy argued that the tactic was necessary from a programming perspective. Amy H. was displeased with both methods since it seemed unreasonable that a cell would “know” if there was enough nutrient for all cells &#40;as the first approach appeared to suggest&#41; nor did it seem appropriate for each cell to individually wait his turn before splitting. <br><br>
                            The meeting concluded with our decision to look into some Swarm examples to get some other ideas for how to proceed and also to determine if it was possible to program things in parallel &#40;so that the cells can do things simultaneously&#41;. The group will meet again on Monday at 3:00 p.m. in a location to be announced.
                        </p>
                      </div>
                        <div role="tabpanel" class="tab-pane" id="juneMinutes">
                        <h5>Meeting Minutes</h5>
                        <p>3 June 2002<br>
                          11&#58;00 a.m. &#45; 1&#58; p.m. HSE&#45;160<br>
                          Cindy Li and Amy H attending<br><br>
                          Li began the meeting by suggesting that we needed to switch tactics given Tony’s comments on Friday. She proposed a single class for the cells with three subclasses &#40;normal, static, and dead&#41; where the subclasses interact with each other. Cindy advised us to start with the normal cells only and add in the static and dead cells later once the program is up and running. When Li and Amy H. asked if that would entail reworking the entire program, Cindy responded that object-oriented programming is designed to allow the addition &#40;and removal&#41; of classes. She then spent some time talking about the heatbug program, and then, having convinced the others that we should begin with the normal subclass of cells only, explained the structure of object-oriented programming using a Swarm tutorial page as a guide.<br><br>
                          Cindy asked what properties we wanted the cells to have, and we agreed that the cells should divide if there’s enough room but not if the tumor itself is too large. Amy H. wanted to be sure that the cells would divide in a way such that the tumor remained somewhat spherical &#40;as opposed to dividing down a line&#41;, and we discussed the grid structure of Swarm. We thought about considering growth in one-dimension initially and adding the second dimension later, but we believed that such an approach would be difficult to implement.<br><br>
                          The idea of using a fixed oxygen gradient was brought up, and we elected to create a class for the environment where we artificially fixed oxygen levels. The goal is for the environment and the tumor to interact with each other in later versions. Cindy recommended that we choose the properties of the cell class and that we split up the work. We determined that the cells should check for available space in four directions &#40up, down, left, and right, and not in diagonal directions&#41;. If space was available, the cell would divide. If more than one space was available, the cell would randomly choose one so that it became unlikely that the tumor would grow in a straight line. The tasks were then divided so that Li and Amy H. would create the classes for the cells and the environment and Cindy would integrate them.</p>

                        </div>
                        <button type="button" class="btn btn-default closeArticle">X</button>
                  </div>
                  </div>
                </div>
              </div>
                <div class="row articleBlock">
                  <div class="col-xs-12 secondSpecialSection">
                    <h4 class="secondSpecialHeader">Simulation Modeling of Epithelial Morphogenesis and Malignancy</h4>
                    <div class="secondSpecialText">
                    <p>
                    Epithelial morphogenesis is a fundamental process of development whereby unorganized single cells give rise to coherent forms of tissues, organs and ultimately a whole organism. During morphogenesis, individual cells collectively exhibit self-organizing behaviors as they undergo cycles of growth, migration and differentiation driven by their genetic program and environmental conditions. An essential, yet poorly understood, aspect of morphogenesis concerns with how the various morphogenic processes of single cells are regulated and coordinated to establish recognizable patterns of multicellularity.<br><br>
                    Currently we are developing a foundational class of computational models to dissect generative principles of cell action that govern epithelial morphogenesis and malignancy in vitro. The developed models are multi-agent systems composed of autonomous in silico cells whose actions are governed by a set of axioms. Our current study focuses on simulation of two types of epithelial cells: &#40;1&#41; Madine-Darby canine kidney &#40;MDCK&#41; cell growth under varying culture conditions, and &#40;2&#41; primary human alveolar type II &#40;AT II&#41; cell alveolization under 3D culture condition.<br><br>
                    Project Contact&#58;Sean H. J. Kim<br><br>
                    Collaborators&#58; Wei Yu, Keith E. Mostov, Michael A. Matthay, Jayanta Debnath</p>
                    <h5>Supplementary Material</h5>
                    <ol type="1" class="sillicoMain">
                      <li><strong>In sillico epithellal analogue &#40;ISEA&#41;</strong><br>
                      <br>
                      <p>Source Codes and supporting files&#58;</p>
                      <ol type="i" class="sillicoSub">
                        <li><a href="http://biosystems.ucsf.edu/epimorph/ISEA_Supp_Mat/ISEA_SIM_README.txt" target="_blank">README</a></li>
                        <li>In silico epithelial analogue &#40;ISEA&#41; source codes <a href="http://biosystems.ucsf.edu/epimorph/ISEA_Supp_Mat/ISEA_SIM_SourceCodes.zip">&#40;download&#41;</a></li>
                        <li>ISEA javadoc<a href="http://biosystems.ucsf.edu/epimorph/ISEA_Supp_Mat/ISEA_SIM_JavaDoc.zip">&#40;download&#41;</a></li>
                        <li>Compiled ISEA jar files<a href="http://biosystems.ucsf.edu/epimorph/ISEA_Supp_Mat/ISEA_SIM.jar">&#40;download&#41;</a></li>
                      </ol>
                      </li>

                      <li><strong>A Computational Approach to Understand in Vitro Alveolar Morphogenisis</strong><br><br>
                      <p>Source Codes and supporting files&#58;</p>
                        <ol type="i" class="sillicoSub">
                          <li><a href="http://biosystems.ucsf.edu/epimorph/ALC_Supp_Mat/README.txt" target="_blank">README</a></li>
                          <li>AT II analogue source codes <a href="http://biosystems.ucsf.edu/epimorph/ALC_Supp_Mat/ATII_Sim_SourceCodes.zip">&#40;download&#41;</a></li>
                          <li>AT II analogue javadoc <a href="http://biosystems.ucsf.edu/epimorph/ALC_Supp_Mat/ATII_Sim_JavaDoc.zip">&#40;download&#41;</a></li>
                          <li>Compiled AT II analogue jar file <a href="http://biosystems.ucsf.edu/epimorph/ALC_Supp_Mat/ATII_SIM.jar">&#40;download&#41;</a></li>
                        </ol>
                      </li>
                      <li><strong>Simulation Modeling of AT II Cell Wound Healing &#40;in preparation&#41;</strong><br><br>
                        <p>Note&#58; simulation space is hexagonal, so when one observes a hexagonal cyst it corresponds to a cross-section through a roundish AT II cyst in vitro.  In case of simulated monolayer repair, the in silico visualization corresponds to the xy-planar view in vitro.</p>
                          <ol type="i" class="sillicoSub">
                            <li><a href="http://biosystems.ucsf.edu/epimorph/ATII_Wound_Videos/ATII_MonoWnd_Random_12hr.mov" target="_blank">Simulated AT II monolayer wound healing ~12 h &#91;random cell movement&#93;</a></li>
                            <li><a href="http://biosystems.ucsf.edu/epimorph/ATII_Wound_Videos/ATII_MonoWnd_Chemo_12hr.mov" target="_blank">Simulated AT II monolayer wound healing ~12 h &#91;chemotaxis&#93;</a></li>
                            <li><a href="http://biosystems.ucsf.edu/epimorph/ATII_Wound_Videos/ATII_MonoWnd_CellDens_12hr.mov" target="_blank">Simulated AT II monolayer wound healing ~12 h &#91;cell density-based movement&#93;</a></li>
                            <li><a href="http://biosystems.ucsf.edu/epimorph/ATII_Wound_Videos/ATII_CystWnd_01s_04.mov" target="_blank">Simulated AT II cyst wound healing &#91;1 wound; 1 cell-width&#93;</a></li>
                            <li><a href="http://biosystems.ucsf.edu/epimorph/ATII_Wound_Videos/ATII_CystWnd_03s_09.mov" target="_blank">Simulated AT II cyst wound healing &#91;3 wounds&#59; 1 cell-width each&#93;</a></li>
                            <li><a href="http://biosystems.ucsf.edu/epimorph/ATII_Wound_Videos/ATII_CystWnd_03t_12.mov" target="_blank">Simulated AT II cyst wound healing &#91;1 wound&#59; 3 cell-widths&#93;</a></li>
                            <li><a href="http://biosystems.ucsf.edu/epimorph/ATII_Wound_Videos/ATII_CystWnd_05s_01.mov" target="_blank">Simulated AT II cyst wound healing &#91;5 wounds&#59; 1 cell-width each&#93;</a></li>
                            <li><a href="http://biosystems.ucsf.edu/epimorph/ATII_Wound_Videos/ATII_CystWnd_05t_06F.mov" target="_blank">Simulated AT II cyst wound healing &#91;1 wound&#59; 5 cell-widths&#93;</a></li>
                          </ol>
                        </li>
                        <li><strong>Kim SHJ, Park S, Yu W, Mostov KE, Matthay MA, Hunt CA. 2007. Systems Modeling of Alveolar Morphogenesis In Vitro.  ISCA 20th International Conference on Computer and Applications in Industry and Engineering &#40;CAINE 2007&#41;, San Francisco, California, USA, November 7-9, 2007</strong><a href="http://biosystems.ucsf.edu/publications/Kim_CAINE2007.pdf" target="_blank"><img class="tinyIcon" src="img/pdf_icon.gif"></a>
                      </ol>
                      <button type="button" class="btn btn-default secondCloseArticle">X</button>
                  </div>

                  </div>
                </div>
                <div class="row articleBlock">
                    <div class="col-xs-12 list-group">
                      <h4 class="list-group-item-heading" id="externalResearchPage"><a href="http://biosystems.ucsf.edu/research_zonation.html" target="_blank">Multi-Agent Model of Hepatic Zonation</a></h4>
                  </div>
                </div>
                  <div class="row articleBlock">
                    <div class="col-xs-12 list-group">
                      <h4 class="list-group-item-heading">MDCK Cystogenesis Simulation Modeling</h4>
                      <div class="list-group-item-text">
                        <p>
                        Epithelial cells perform essential functions throughout the body, acting as both barrier and transporter and allowing an organism to survive and thrive in varied environments. Although the details of many processes that occur within individual cells are well understood, we still lack a thorough understanding of how cells coordinate their behaviors to create complex tissues. In order to achieve deeper insight, we created a list of targeted attributes and plausible rules for the growth of multicellular cysts formed by Madin-Darby canine kidney &#40;MDCK&#41; cells grown in vitro. We then designed in silico analogues of MDCK cystogenesis using object-oriented programming. In silico components &#40;such as the cells and lumens&#41; and their behaviors directly mapped to in vitro components and mechanisms. We conducted in vitro experiments to generate data that would validate or falsify the in silico analogues and then iteratively refined the analogues to mimic that data. Cells in vitro begin to stabilize at around the fifth day even as cysts continue to expand. The in silico system mirrored that behavior and others, achieving new insights. For example, luminal cell death is not strictly required for cystogenesis, and cell division orientation is very important for normal cyst growth.<br><br>
                        Project Contact&#58; Jesse A. Engelberg<br><br>
                        Collaborators&#58; Anirban Datta, Keith E. Mostov
                      </p>
                      <h5>In Silico MDCK Analogue Supplementary Material&#58;</h5>
                      <ol type="i">
                        <li>Source code <a href="http://biosystems.ucsf.edu/mdcksim/ISMA.zip">&#40;download&#41;</a></li>
                        <li><a href="http://biosystems.ucsf.edu/mdcksim/README.txt" target="_blank">User manual</li>
                      </ol>
                    </div>
                    </div>
                  </div>
              </div>
            </div>
          </div>
        </div>

          <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
          <!-- Latest compiled and minified JavaScript -->
          <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>

          <script src="js/research.js"></script>
      </body>
    </html>
